src/psyclone/domain/lfric/lfric_constants.py

```diff
@@ -62,13 +63,13 @@ def __init__(self):
         LFRicConstants.HAS_BEEN_INITIALISED = True
         api_config = Config.get().api_conf("dynamo0.3")
```

Comment:
Please could you disable the pylint 'too-many-statements' warning for this constructor.

Reply:
Disabled.